### ISSUE TITLE
Documentation - versionadded directive rendered differently between RST and Markdown syntax

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#8742]
## Summary 

The `versionadded` tag output is different between RST and Markdown versions of the documentation.

This creates an inconstancy between the two variations and should be resolved with either a preference to convention, styling fixes or maybe we are using the myst directive incorrectly (maybe it should be a role). 

See - https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html

## Details 

MD:
![md](https://user-images.githubusercontent.com/31622/175512160-4e629720-04e9-4c60-ab62-b398c3495bbc.png)

RST:
![rst](https://user-images.githubusercontent.com/31622/175512146-071634f8-1e8a-4813-b3bc-5430f3b9bcb7.png)
d

_Originally posted by @zerolab in https://github.com/wagtail/wagtail/pull/8568#pullrequestreview-1018264953_

**Comment by zerolab** [comment]
Thank you @lb-,

I've been digging more into this and after a fresh requirements install, it looks like the styles are the expected admonition-y ones, rather than just a plain label.

Can only think of either some heavily cached styles on my side, or perhaps a fluke in building the docs.

Worth checking on your side to confirm and perhaps we can close this.

**Comment by lb-** [comment]
@zerolab  ahh yep - must have been an odd caching thing locally on yours, I find that busting the docs cache locally is really difficult.

Here is a comparison of the same `versionadded` admonition but the top one is in RST and the bottom is in MD - they render the same.

Closing as non-issue.


<img width="1645" alt="Screen Shot 2022-06-25 at 10 49 40 pm" src="https://user-images.githubusercontent.com/1396140/175774217-da4b52dc-3b2c-41f8-8e2a-15ca1f597afe.png">


However, I did find a link that is not rendering correctly `Levenshtein edit distance` - so I will fix that up :|
